source/crud.txt

```diff
@@ -31,7 +31,19 @@ CRUD Operations
    Compound Operations </crud/compound-operations>
    Transactions </crud/transactions>
    Configure CRUD Operations </crud/configure>
+   Generate Custom _id Values </crud/pkFactory>
    Store Large Files </crud/gridfs>
 
 CRUD (Create, Read, Update, Delete) operations enable you to work with
-data stored in MongoDB.
\ No newline at end of file
+data stored in MongoDB.
+
+- :ref:`node-insert`
+- :ref:`node-query`
+- :ref:`node-update`
+- :ref:`node-delete`
+- :ref:`node-bulk-write`
+- :ref:`node-crud-compound-operations`
+- :ref:`node-transactions`
+- :ref:`node-configure`
+- :ref:`node-pkfactory`
```

Comment:
Not edited in this file, but can you take away the monospace from _id in the "Generate Custom _id Values"  page title? 
Style guide: https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#:~:text=Don%27t%20apply%20font%20treatments%20(bold%2C%20italics%2C%20or%20monospace)%20to%20text%20in%20a%20title%20or%20heading.